English blog post (#241)
yabellini/SitioAcademico: content/blog/2024-07-28-rOpenSci-2years-Writing-Speaking-Teaching-en/index.md

---
title: "Project 3 - Two Years of Writing, Speaking, and Teaching with the rOpenSci Community"
author: Yanina Bellini Saibene
summary: "One of the core competencies of a Community Manager is Communication. The third 'project' I will tell you about is not a project per se but a series of activities related to communication and outreach." 
date: '2024-07-28'
categories:
  - Community
  - 100DaysToOffload
  - rOpenSci
tags:
  - Community
  - 100DaysToOffload
  - rOpenSci
---

{{< figure src="featured.jpg" alt="A woman writing in withe paper">}}

> > Foto de <a href="https://unsplash.com/es/@ana_tavares?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Ana Tavares</a> en <a href="https://unsplash.com/es/fotos/mujer-escribiendo-en-papel-blanco-VDwINWBdX0Y?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>

Author: ```suggestion
> Picture by <a href="https://unsplash.com/es/@ana_tavares?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Ana Tavares</a> on <a href="https://unsplash.com/es/fotos/mujer-escribiendo-en-papel-blanco-VDwINWBdX0Y?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
```

rOpenSci also has a [newsletter](https://ropensci.org/tags/newsletter/). If you are subscript and enjoy the content, [Maëlle Salmon](https://ropensci.org/author/ma%C3%ABlle-salmon/) is the main responsible for curating the content, publishing, and making sure our newsletter is in your inbox every month.

The rest of the staff contributes to the HQ section, where we mention the most important news of the month about the projects we lead, along with content suggestions for the different sections of the newsletter. Since I joined rOpenSci, I have contributed to **26 newsletters**.

## Books, guides, and tip sheets.

One of my favorite things about rOpenSci is the culture of openness it preaches and practices. We document our processes and share them openly in books, guides, or tip sheets for other people and organizations.

> In these two years, I have co-author guides for the champions program project in a [Play Book](https://ropenscilabs.github.io/champions-program-playbook/) on how to organize and run the program, [the Mentor Guidelines](https://ropensci-org.github.io/champions-mentor-guidelines/) for the people that will mentoring our champions, and a [translation guide](https://translationguide.ropensci.org/) for our multilingual publishing project.

Author: ```suggestion
In these two years, I have co-authored two books for the champions program: a [Play Book](https://ropenscilabs.github.io/champions-program-playbook/) on how to organize and run the program and the [mentor guidelines](https://ropensci-org.github.io/champions-mentor-guidelines/) for the people who will mentor our champions.  I also co-wrote a [localization and translation guide](https://translationguide.ropensci.org/) for our multilingual publishing project.
```

I contribute by updating or adding new content to existing guides and books, such as the [rOpenSci Community Contributing Guide](https://contributing.ropensci.org/) or the [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/).

## Talks

Some community managers are the visible faces of the community, so part of our job is to give talks in different places and for different audiences.

I enjoy giving talks, explaining what we do, and sharing what I have learned, so I will generally say yes to an invitation to give a talk. Especially if the event is online, it is very easy for me to participate. I also send proposals to events and conferences. 

Since I became rOpenSci Community Manager, I have delivered **43 talks** related to or representing rOpenSci. [I shared the video and slides when available.](/talk/) Half of the talks were in Spanish, the other half in English.

> {{< figure src="charlas.jpg" alt="Yani's pictures giveing talks and differents Yani's talk flyers">}}

Author: ```suggestion
{{< figure src="charlas.jpg" alt="Yani's pictures giving talks and differents Yani's talk flyers">}}
```

I got invitation to share about rOpenSci with working groups, committees, and boards of directors like [Bioconductor](/talk/2023_bioconductor/), [Instituto Nacional de Tecnologia Agropecuaria](/talk/2022_ropensci_pae_inta/), [Euro-BioImaging](/talk/2023_eurobioimaging/), and [Tampa Bay Estuary Program ](/talk/2023_tbep/). I also participated as a speaker, panelist, or moderator in other community events, like [CS&S](/talk/2022_css/), [CSCCE](/talk/2022_cscce_cc_nov/), [The Carpentries](/talk/2023-10-30-thecarpentries25/), [GitHub](/talk/2022_octogatos/), [R-Ladies Chapters](/talk/2023_r-ladiesmexico/), and [R User Groups](/talk/2022_ropensci_champion_abuja/).

## Training

Teaching is one of my favorite activities. As part of the Champions Program Project, I design and then deliver [training on mentoring](https://ropensci-training.github.io/ropensci-mentors/), [R package development best practices](https://paocorrales.github.io/git-developing-software-together/), and [community management skills](https://osschampionsprogram.netlify.app/#/title-slide).

> I have had the chance to teach and assist in 22 workshops for rOpensci with talented members of our community: [Maelle Salmon, Paola Corrales, Mauro Lepore, Heather Turner, and CSCCE staff](/blog/2023/08/04/champions-program-training/). Open Life Science invited me to one of their training sessions to share about [testing and coding styles](/talk/2023_ols/). 

Author: ```suggestion
I have had the chance to teach and assist in **22 workshops** for rOpensci with talented members of our community: [Maelle Salmon, Paola Corrales, Mauro Lepore, Heather Turner, and CSCCE staff](/blog/2023/08/04/champions-program-training/). Open Life Science invited me to one of their training sessions to share about [testing and coding styles](/talk/2023_ols/). 
```
